lib/DriverTool/swift_symbolgraph_extract_main.cpp

```diff
@@ -162,19 +162,18 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args,
     }
   }
 
-  symbolgraphgen::SymbolGraphOptions Options{
-      OutputDir,
-      Target,
-      ParsedArgs.hasArg(OPT_pretty_print),
-      AccessLevel::Public,
-      !ParsedArgs.hasArg(OPT_skip_synthesized_members),
-      ParsedArgs.hasArg(OPT_v),
-      ParsedArgs.hasArg(OPT_skip_inherited_docs),
-      ParsedArgs.hasArg(OPT_include_spi_symbols),
-      /*IncludeClangDocs=*/false,
+  symbolgraphgen::SymbolGraphOptions Options;
+  Options.OutputDir = OutputDir;
+  Options.Target = Target;
+  Options.PrettyPrint = ParsedArgs.hasArg(OPT_pretty_print);
+  Options.EmitSynthesizedMembers = !ParsedArgs.hasArg(OPT_skip_synthesized_members);
```

Comment:
Small nit on the naming here; the 'protocol implementations' one is exclusive (skip) and this one inclusive (emit).